daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -462,7 +462,8 @@ You can configure pub/sub to publish or consume data encoded using [Avro binary
 ### Configuration
 
 {{% alert title="Important" color="warning" %}}
-Currently, only message value serialization/deserialization is supported. Since cloud events are not supported, the `rawPayload=true` metadata must be passed.
+Currently, only message value serialization/deserialization is supported. Since cloud events are not supported, the `rawPayload=true` metadata must be passed when publishing Avro messages.
+Please note that `rawPayload=true` should NOT be set for consumers, as the message value will be wrapped into a CloudEvent and base64-encoded. Leaving `rawPayload` as default, will send the Avro-decoded message to the application.
```

Comment:
To reduce ambiguity, we can say "Leaving the default value of `rawPayload=false` will..."